src/main/java/com/raoulvdberge/refinedstorage/screen/grid/GridScreen.java

```diff
@@ -559,4 +576,8 @@ public void updateScrollbar() {
     public static IGridSorter getDefaultSorter() {
         return new NameGridSorter();
     }
+
+    public boolean avoidSorting() {
+        return hasShiftDown() && !doSort;
```

Comment:
Can this `hasShiftDown` check be moved to the place where `doSort` is modified?

Reply:
And perhaps rename this to `canSort`.

Reply:
As `doSort` gets modified in `onMouseClick` moving `hasShiftDown` there introduces multiple edge cases.